setup.py

```diff
@@ -3,7 +3,7 @@
 import sys, os
 
 
-version = '2.6.4'
+version = '2.39'
```

Comment:
IMHO this doesn't make sense. If you keep following Testdroid version what if you need to do bug fix to api client during one version. is going  to be 2.39.1 (at least the orig version should be then 2.39.0) other thing what if there is no update to client does it jump from 2.39 to 2.58 for example?